test/tbb/test_global_control.cpp

```diff
@@ -272,3 +273,30 @@ TEST_CASE("test concurrent task_scheduler_handle destruction") {
     stop = true;
     thr1.join();
 }
+
+//! \brief \ref regression
+TEST_CASE("Thread should sleep when soft_limit is zero") {
```

Comment:
Shouldn't this test be moved under `conformance_global_control.cpp`? It seems like this test verifies compliance with the specification:

> With max_allowed_parallelism set to 1, global_control enforces serial execution of all tasks by the application thread(s), that is, the task scheduler **does not allow** worker threads to run. There is one exception: if some work is submitted for execution via task_arena::enqueue, a single worker thread will still run ignoring the max_allowed_parallelism restriction.

Reply:
Done